### PR TITLE
__init__ method: changed signal_scaling if statement

### PR DESCRIPTION
**Pull request by josephsmann**
line 289: signal_scaling in [0, 1, (0,1)]    
admits False which made the following False condition unreachable. 
(my first PR please inform liberally re: protocol etc)

also added the
import nibabel as nib

### CODE REVIEW

**Comment by bthirion** [comment]
Nice, thx !
Can you add the corresponding test to the test suite to ensure that the correct behavior is enforced ?

File: nistats/first_level_model.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 import numpy as np
 import pandas as pd
+import nibabel as nib
```
Comment (bthirion) [inline nistats/first_level_model.py]: Rather than this import, could you directly use Nifti1Image imported at the line below ?

**Comment by josephsmann** [comment]
@bthirion I have attempted what you requested re: the import statement and the test function. Comments will be most welcome - my OSS experience is minimal.

File: nistats/tests/test_first_level_model.py [hunk 2]
```diff
@@ -382,3 +382,28 @@ def test_first_level_models_from_bids():
...
+    first_level_model.fit(fmri_data, design_matrices=design_matrices)
+    # trivial test of signal_scaling value
+    assert_true(first_level_model.signal_scaling==False)
+    # assert that our design matrix has one constant
+    assert_true(first_level_model.design_matrices_[0].equals(
+                    pd.DataFrame([1.0, 1.0], columns=['a'])))
+    # assert that we only have one theta as there is only on voxel in our image
+    assert_true(first_level_model.results_[0][0].theta.shape == (1,1))
+    # assert that the theta is equal to the one voxel value
+    assert_almost_equal(first_level_model.results_[0][0].theta[0,0],6.0,2)
```
Comment (bthirion) [inline nistats/tests/test_first_level_model.py]: The test is great. I believe that you could have made it more elementary, i.e. testing only the function you've changed, but that's fine.
There are some PEP8 issues (space around operators). Can you fix them ?
Thx again !

**Comment by bthirion** [comment]
LGTM, thx. let's wait a little bit for another review...

**Comment by bthirion** [comment]
Thx again.

**Comment by josephsmann** [comment]
my pleasure.

**Pull request merged**